gke/config.yaml

```diff
@@ -123,16 +123,15 @@ credentials:
   serviceAccountKeyfile: '{{.GKE_SA_KEYFILE_2}}'
   defaults:
     namespace: {{.GKE_DEFAULT_NAMESPACE_2}}
-    certificatesecret: {{.GKE_DEFAULT_CERTIFICATE_2}}
```

Comment:
Fine to roll back now for sake of speed, but let's fix the shared secret for this environment asap, so we can retest.